vowpalwabbit/reductions/cb/cb_explore.cc

```diff
@@ -427,8 +426,10 @@ base_learner* cb_explore_setup(VW::setup_base_i& stack_builder)
   auto* l = make_reduction_learner(
       std::move(data), base, learn_ptr, predict_ptr, stack_builder.get_setupfn_name(cb_explore_setup) + name_addition)
                 .set_params_per_weight(params_per_weight)
+                .set_output_prediction_type(VW::prediction_type_t::multiclass)
```

Comment:
This conflicts with the line just below

Reply:
Good catch! meant input_prediction_type here